contributing-guides/style-guide.md

```diff
@@ -93,7 +103,15 @@ Due to [various compatibility differences](https://learn.microsoft.com/powershel
 > More information: <https://learn.microsoft.com/powershell/module/microsoft.powershell.management/clear-recyclebin>.
 ```
 
-## Aliases
+## <a name="pages"></a>Pages
+
+### Platform differences
+
+If you are afraid the commands may differ between platforms or operating systems (e.g. Windows vs macOS), most [tldr pages clients](https://github.com/tldr-pages/tldr/wiki/tldr-pages-clients) will choose the most suitable version of the command.
+
+In this case, the information of the Windows version of `cd` (stored in `pages/windows/cd.md`) will be displayed by default to Windows users, and a generic/common version (stored in `pages/common/cd.md`) will be displayed for Linux, macOS, and other platforms.
```

Comment:
I didn't edit this text, I just separated it from the "Imperative Mood" section because it didn't fit there, it almost looked like a random text in the middle of the section.

Reply:
Okay, I see. But considering your PR in general the diff is quite extensive, is it just table of content headings? The diff has gone a bit mad.

Reply:
I have currently rebased this PR to accommodate the recent changes to the style guide, I will provide a detailed changelog once my changes are complete.